src/ashley/models.py

```diff
@@ -0,0 +1,51 @@
+"""Declare the models related to ashley ."""
+
+from django.conf import settings
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from lti_provider.models import LTIConsumer
+
+
+class UserInfo(models.Model):
```

Comment:
Have you thought of declaring a custom User with added fields? It's the common practice if what you want is to add the remote_identifier (the username field is already on Django's user).

Reply:
By «_custom User with added fields_», you mean a custom User model ?

Like I said in the docstring of this class, I did this model to avoid this. Because according to the [official documentation](https://docs.djangoproject.com/en/3.0/topics/auth/customizing/#reusable-apps-and-auth-user-model), it is not recommended to extend to implement a custom User model for reusable apps.

The big question is : should we consider `ashley` as a **reusable app**, or a **project** ?
For the moment, I try do design it as a reusable app. WDYT ?

Reply:
But it could be an abstract model that people can override... so it is even better than your model because they can add their own fields and/or change the fields you added? I think I saw that django machina is doing it for all their models.

Reply:
For all models it's a good idea, except for the `User` model :
_«A project may use many apps, and two reusable apps that implemented a custom user model couldn’t be used togethe»_